bin/_blackbox_common.sh

```diff
@@ -277,6 +277,9 @@ function shred_file() {
     #NOTE: srm by default uses 35-pass Gutmann algorithm
     CMD=srm
     OPT=-f
+  elif _F=$(mktemp); rm -P "${_F}" >/dev/null 2>/dev/null ; then
```

Comment:
Add "--" before the filename so that people can safely delete a file called "-r".

```
rm -P "${_F}"
changes to:
rm -P -- "${_F}"
```